rawspec.c

```diff
@@ -192,26 +194,43 @@ char tmp[16];
   int input_conjugated = -1;
   int only_output_ics = 0;
 
+  // Selected dynamic debugging
+  int flag_debugging = 0;
+
+  // FBH5 fields
+  int flag_fbh5_output = 0;
+
   // For net data rate rate calculations
   double rate = 6.0;
   double sum_inv_na;
   uint64_t total_packets = 0;
   uint64_t total_bytes = 0;
   uint64_t total_ns = 0;
 
+  // Show librawspec version on startup
+  printf("rawspec using librawspec %s\n", rawspec_version_string());
+
   // Init rawspec context
   memset(&ctx, 0, sizeof(ctx));
   ctx.Npolout[0] = 1; // others will be set later
 
   // Parse command line.
   argv0 = argv[0];
-  while((opt=getopt_long(argc,argv,"a:b:d:f:g:HI:i:n:o:p:r:Ss:t:hv",long_opts,NULL))!=-1) {
+  while((opt=getopt_long(argc, argv, "a:b:d:f:g:HSjzs:i:n:o:p:r:t:hv", long_opts, NULL)) != -1) {
```

Comment:
Not an issue for me, but why collect all of the flags (as opposed to options) together?

Reply:
Yes, I fixed that boo-boo.

Reply:
E.g. https://www.gnu.org/software/libc/manual/html_node/Getopt-Long-Option-Example.html

Reply:
Sure. So grouping them all is just a preference? The previous version seemed to embody to the above protocol. No mind in that case.

Reply:
They have to be grouped with one other option.  Doesn't matter which one.